src/components/Bookmarks/Bookmarks.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import bookmarkActive from './../../images/red_heart.png'
+import bookmarkInactive from './../../images/rosa_heart.png'
+
+Bookmarks.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  isBookmarked: PropTypes.bool,
+  children: PropTypes.node,
+}
+
+export default function Bookmarks({ isBookmarked, onClick, ...props }) {
+  return (
+    <BookmarkStyled {...props}>
+      <img
+        src={isBookmarked ? bookmarkActive : bookmarkInactive}
```

Comment:
I would rename the import name to something like redHeart & rosaHeart since I know already by the `?`behind isBookmarked that we are dealing with an active or inactive bookmark. Therefore a descriptive name describing what gets shown upon being active could work better here